src/jsonGraphToCypher.js

```diff
@@ -71,9 +68,9 @@ const linkToEdgeLabel = (link) => {
 const isTypeNode = (node, links) => {
   const [link] = links.filter((link) => link.target === node.id);
   if (link?.label === 'type') {
-    return true;
+    return [true, link];
   }
-  return false;
+  return [false, link];
```

Comment:
This works but the return value and the function name are sort of different. nit: getLinkId and isTypeNode separate function

Reply:
Yeah that's cool